TCAT/Views/RouteDiagramSegment.swift

```diff
@@ -51,7 +51,10 @@ class RouteDiagramSegment: UIView {
             return self.icon is UIImageView ? spaceBtnWalkIconAndSuperview : (self.icon is BusIcon ? 0 : 8.5)
         }
         let spaceBtnStopDotCenterXAndStopLabel: CGFloat = 26
-
+        let screenWidth = UIScreen.main.bounds.width
+        let stopLabelRightPadding: CGFloat = 16
+        let stopLabelYPosition: CGFloat = 101
+        let stopLabelWidth: CGFloat = screenWidth - stopLabelYPosition - stopLabelRightPadding
```

Comment:
should we be subtracting `stopLabelYPosition` or `stopLabelXPosition` since its width?